tutorials/3d/softbody.rst

```diff
@@ -0,0 +1,64 @@
+.. _doc_softbody:
+
+Soft body
+=========
+
+Soft body simulates cloth movement and other deformable bodies.
```

Comment:
Soft bodies (or *soft-body dynamics*) simulate movement, changing shape and other physical properties of deformable objects.
This can for example be used to simulate cloth or to create more realistic characters.